sci-physics/vgm/vgm-5.2.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vmc-project/${PN}.git"
+else
+	MY_PV=$(ver_rs 1- -)
+	SRC_URI="https://github.com/vmc-project/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+DESCRIPTION="Virtual Geometry Model for High Energy Physics Experiments"
+HOMEPAGE="https://github.com/vmc-project/vgm/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc examples +geant4 +root test"
+
+RDEPEND="
+	sci-physics/clhep:=
+	geant4? ( >=sci-physics/geant-4.11:=[c++17] )
+	root? ( sci-physics/root:=[c++17] )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[dot] )
```

Comment:
doxygen is a BDEPEND here (its tool is just called during the build).

Reply:
Great catch, thanks! Fixed in the last force-push both for the new version and `9999`. 
It's probably not useful backporting this fix to the older versions in-tree, or is it?

Reply:
No problem! Nah, I wouldn't worry about it.